specification.md

```diff
@@ -1910,26 +1910,25 @@ we can use this expression in the workflow "version" parameter:
 
 ```json
 {
-   "id": "MySampleWorkflow",
-   "name": "Sample Workflow",
+   "name": "MySampleWorkflow",
+   "description": "Sample Workflow",
    "version": "${ .inputVersion }",
    "specVersion": "0.8"
 }
 ```
 
 which would set the workflow version to "1.0.0".
-Note that the workflow "id" property value is not allowed to use an expression. The workflow
-definition "id" must be a constant value.
+Note that the workflow "name" property value is not allowed to use an expression. The workflow
+definition "name" must be a constant value.
 
 ### Workflow Definition Structure
 
 | Parameter | Description | Type | Required |
 | --- | --- |  --- | --- |
-| id | Workflow unique identifier | string | yes (if `key` is not defined) |
-| key | Domain-specific workflow identifier | string | yes (if `id` is not defined) |
-| name | Workflow name | string | no |
+| name | The name that identifies the workflow definition, and which, when combined with its version, forms a unique identifier. | string | yes |
+| version | Workflow version. MUST respect the [semantic versioning](https://semver.org/) format. If not provided, latest is assumed | string | no |
```

Comment:
Could you put latest inside a ` MD block, so that it looks like this: `latest`. 

It is imho more readable, and allows users to quickly identify "tagged" values.